modules/mediawiki/files/bin/deploy-mediawiki.py

```diff
@@ -110,7 +110,7 @@ def run(args, start):
         serverlist = ['mw8', 'mw9', 'mw10', 'mw12', 'mw13', 'mwtask1']
         sync = True
     elif args.servers == 'scsvg':
-        serverlist = ['mw101', 'mw102', 'mw111', 'mw112', 'mw121', 'mw122', 'mw131', 'mw132']
+        serverlist = ['mw101', 'mw102', 'mw111', 'mw112', 'mw121', 'mw122']
```

Comment:
What about mwtask101?

Reply:
It's the canary. Same as mw11 not being in the list for ovlon

Reply:
Oh, right!